ssz/sedes/base.py

```diff
@@ -69,6 +69,14 @@ def deserialize(self, data: bytes) -> TDeserialized:
     def hash_tree_root(self, value: TSerializable) -> bytes:
         pass
 
+    @abstractmethod
+    def serialize_text(self, value: TSerializable) -> Any:
```

Comment:
One thing I feel uncomfortable of this PR is this optional feature requires modifications to all these core classes. Another thing is not sure if `serialize_text` or `deserialize_text` are the right words to call these methods. They worked together to produce some texts, but individually only output some dictionaries or lists.